src/pseudopeople/configuration/__init__.py

```diff
@@ -1,2 +1,2 @@
-from pseudopeople.configuration.entities import Keys
+from pseudopeople.configuration.entities import DEFAULT, NO_NOISE, Keys
```

Comment:
Do we need to expose DEFAULT here? This would minimize my concern of a user trying to do something like `get_config(user_config=DEFAULT)`